prowler

```diff
@@ -83,12 +84,19 @@ USAGE:
                             (i.e.: ProwlerRole)
       -T                  session durantion given to that role credentials in seconds, default 1h (3600) recommended 12h, requires -R and -T
                             (i.e.: 43200) 
+      -w                  whitelist file. (Lines starting with # are ignored as comments) Format:
```

Comment:
```file prowler: Line 174 could cause problems if CSV or json is used, am I right? I don 't think it is needed.```

@toniblyx yeah, I was hoping that this help doc clarifies the structure we're relying on here, unless I am missing something else about the CSV/JSON concern.

Reply:
Overall, what do you think about migrating this project to a python/go etc to help simplify some of the abstractions? Just a thought - you might get more contribs and somewhat easier code - I noticed that the shell code in here has a lot of warnings according to shell-check.
It's def your preference as the project owner - hope this doesn't come across the wrong way :)